test_utils/factories.py

```diff
@@ -127,3 +127,20 @@ class Meta(object):
 
     enterprise_customer = factory.SubFactory(EnterpriseCustomerFactory)
     provider_id = factory.LazyAttribute(lambda x: FAKER.slug())
+
+
+class EnterpriseCustomerBrandingFactory(factory.django.DjangoModelFactory):
+    """
+    EnterpriseCustomerBrandingFactory factory.
+    Creates an instance of EnterpriseCustomerBrandingFactory with minimal boilerplate - uses this class' attributes as
+    default parameters for EnterpriseCustomerBrandingFactory constructor.
+    """
+
+    class Meta(object):
+        """
+        Meta for EnterpriseCustomerBrandingFactory.
+        """
+
+        model = EnterpriseCustomerBrandingConfiguration
+
+enterprise_customer = factory.SubFactory(EnterpriseCustomerFactory)
```

Comment:
@asadiqbal08 I think this statement is not indented correctly, shouldn't it be part of `EnterpriseCustomerBrandingFactory ` class ?